### PR TITLE
[Doc Improvement][Update task module to dialog for Webhooks and cards files][3480220]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/40ffe2b283ccfdc9fe8dde4ba91f67df626a8bf3/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Finvoking-task-modules.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ef55b9f4-b053-487f-88a5-995cf099c466) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/45d70f59eeffa0418b85bfa54eea6da5b7ba8371/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Finvoking-task-modules.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f2b6cd0f-2390-4e21-87d6-56fa8657ae4c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/45d70f59eeffa0418b85bfa54eea6da5b7ba8371/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d76a7bde-b503-4e7d-ba90-3dc580320ab3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dc775579b9363ae8e4ba2ddb86805a41bc6a3eb6/msteams-platform%2Fapps-in-teams-meetings%2Fmeeting-apps-apis.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7f51548d-23c3-41be-a484-a7592e69a19b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dc775579b9363ae8e4ba2ddb86805a41bc6a3eb6/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Finvoking-task-modules.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4dc759a-9d11-4bbc-9ac4-9db215c2bb0e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dc775579b9363ae8e4ba2ddb86805a41bc6a3eb6/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-bots.md) | 89 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da1aa3f7-0307-466a-91ac-de6702bdf385) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dc775579b9363ae8e4ba2ddb86805a41bc6a3eb6/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e1b3ad19-5f4b-485b-89f4-74f0396a5ad4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/what-are-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dc775579b9363ae8e4ba2ddb86805a41bc6a3eb6/msteams-platform%2Ftask-modules-and-cards%2Fwhat-are-task-modules.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c7c6a8f9-ceef-4782-8092-25b74caca33e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6edd5bbda138ffe561df72ec07a6cfa6a176bb1d/msteams-platform%2Fapps-in-teams-meetings%2Fmeeting-apps-apis.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/722ef186-0e27-43c3-b1cd-e8f9997a08b6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6edd5bbda138ffe561df72ec07a6cfa6a176bb1d/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Finvoking-task-modules.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4c57c326-08f9-4d76-9f0a-2f719bc40886) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6edd5bbda138ffe561df72ec07a6cfa6a176bb1d/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-bots.md) | 89 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/91caaa2b-ad51-4025-8698-df4ba9599dc9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6edd5bbda138ffe561df72ec07a6cfa6a176bb1d/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e56966fc-624e-4f1b-aeb1-d416f6279471) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/what-are-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6edd5bbda138ffe561df72ec07a6cfa6a176bb1d/msteams-platform%2Ftask-modules-and-cards%2Fwhat-are-task-modules.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed920084-139f-4ded-8aae-2c19b6a83e37) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md [hunk 1]
```diff
@@ -1,62 +1,62 @@
 ---
-title: Invoke and dismiss task modules
-description: Learn about invoking and dismissing task modules, task info object, task module sizing, task module deep link syntax using Code samples
+title: Invoke and dismiss dialogs
+description: Learn about invoking and dismissing dialogs (task modules), task info object, dialog (task module) sizing, dialog (task module) deep link syntax using Code samples
```
[the remark this answers is not in the document]
Comment (v-npaladugu) [reply msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md]: Updated

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md](https://github.com/MicrosoftDocs/msteams-docs/blob/abb520876f33a45a32f938dfdb8f80b9362ae75b/msteams-platform%2Fapps-in-teams-meetings%2Fmeeting-apps-apis.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/33699745-4705-4b87-b020-f9e94aad0bf6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/abb520876f33a45a32f938dfdb8f80b9362ae75b/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Finvoking-task-modules.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/10cb603d-f909-4bcd-9ced-5e37790a7080) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/abb520876f33a45a32f938dfdb8f80b9362ae75b/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-bots.md) | 89 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d635da0-5f22-420f-8a38-c901133522e1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/abb520876f33a45a32f938dfdb8f80b9362ae75b/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/277abe5e-7865-4975-b949-95b86c289c3a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/what-are-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/abb520876f33a45a32f938dfdb8f80b9362ae75b/msteams-platform%2Ftask-modules-and-cards%2Fwhat-are-task-modules.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0c48fb2-ce6d-4bdf-ad3e-d454adfca8b8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c413e7f6f2928e452faa585f349091a1fbc95827/msteams-platform%2Fapps-in-teams-meetings%2Fmeeting-apps-apis.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/49635ffe-e1a2-4f41-ae8d-b3bcb731622b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c413e7f6f2928e452faa585f349091a1fbc95827/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Finvoking-task-modules.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e43ec877-ac0d-4d2e-af2c-68a4b9143f54) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c413e7f6f2928e452faa585f349091a1fbc95827/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-bots.md) | 89 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c8a8870f-bcc0-4dc7-8c07-1305c0e8fe08) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c413e7f6f2928e452faa585f349091a1fbc95827/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c4c3d8e6-0834-4e7b-9d4a-6b2186455ff1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/what-are-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c413e7f6f2928e452faa585f349091a1fbc95827/msteams-platform%2Ftask-modules-and-cards%2Fwhat-are-task-modules.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7ee2da5c-b29d-4dd3-80f8-bc66ab880d6e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6fbb919d39f33ad2308b11ef17edc73a09c74b5e/msteams-platform%2Fapps-in-teams-meetings%2Fmeeting-apps-apis.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fc258432-5500-46b5-b199-0652cf2c6120) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6fbb919d39f33ad2308b11ef17edc73a09c74b5e/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Finvoking-task-modules.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed806993-3472-4543-b7e2-90c72297334d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6fbb919d39f33ad2308b11ef17edc73a09c74b5e/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-bots.md) | 89 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/97b3de8a-983f-46c3-9292-dcb0eef67e81) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6fbb919d39f33ad2308b11ef17edc73a09c74b5e/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/66752a2c-3117-4b76-9752-7769dc76b058) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/what-are-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6fbb919d39f33ad2308b11ef17edc73a09c74b5e/msteams-platform%2Ftask-modules-and-cards%2Fwhat-are-task-modules.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5d0580f0-ed8a-444b-ae75-506fafe6c1dc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**